### PR TITLE
Defining typescript interface for options of method to make screenshots

### PR DESCRIPTION
**Pull request by Dok11**
From https://forum.babylonjs.com/t/define-typescript-interface-for-options-of-method-to-make-screenshots/4713
All test are passed and build successfully complete

### CODE REVIEW

**Comment by sebavan** [comment]
Could the interface be moved within the screenshotTools file ? This helps preventing import from tools in ES6 when not necessary.

Thx

**Comment by Dok11** [comment]
@sebavan check it please https://github.com/BabylonJS/Babylon.js/pull/6630/commits/6b367047b53745174d8c4ac9a6687c42c6e67070

**Comment by sebavan** [comment]
LGTM :-) Could you add an entry in dist/preview release/what's new.md ? and all the build will be green and ready to go in.

**Comment by Dok11** [comment]
> add an entry in dist/preview release/what's new.md ?

Seems I cant understand.. it it written here what I must not commit dist/preview
https://doc.babylonjs.com/how_to/how_to_start#do-not-commit

**Comment by sebavan** [comment]
This is utterly confusing I agree and I will fix the doc ASAP, the what s new file is the only one that needs to be updated. This is our base to create the changelog for the next release.

File: src/Misc/tools.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { PromisePolyfill } from './promise';
 import { TimingTools } from './timingTools';
 import { InstantiationTools } from './instantiationTools';
 import { GUID } from './guid';
+import { IScreenshotSize } from './screenshotTools';
```
Comment (deltakosh) [inline src/Misc/tools.ts]: The only problem is that we do not want to import the file automatically for size reason (treeshacking). Do you mind moving the interface in its own file?

**Comment by Dok11** [comment]
> Do you mind moving the interface in its own file?

I wanted itm but cant understand system for defining interfaces. In my projects I usual define interfaces in self directory called `interfaces` and use only one `export` construction per one file.
But here I dont see similar system or any other. So I make `interfaces` folder in the `/Misc`.

**Comment by deltakosh** [comment]
iInspectable is already in the misc folder so it is ok to drop that file there as well
no need for a new folder

**Comment by Dok11** [comment]
> iInspectable is already in the misc folder so it is ok to drop that file there as well
no need for a new folder

But in this folder already have many interfeces, these may be moved into `interfaces` folder. I dont did it because my PR not about it and not about refeactoring

**Comment by deltakosh** [comment]
I agree but let's do it later then. (Also you have a doc issue)

**Comment by Dok11** [comment]
> Also you have a doc issue

I thinked documentation generated automatic, it is not right? Or you about `what's new.md` file?

**Comment by Dok11** [comment]
I updated file `what's news`. Is now all ok?

**Comment by deltakosh** [comment]
You can see the checks here: https://github.com/BabylonJS/Babylon.js/pull/6630/checks

The documentation problem: "[18:09:46] Missing text for Interface : IScreenshotSize (id: 24186) babylon.d.ts:31576"

File: src/Misc/interfaces/screenshotSize.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export interface IScreenshotSize {
```
Comment (deltakosh) [inline src/Misc/interfaces/screenshotSize.ts]: You need to add a comment to explain the Interface

**Comment by Dok11** [comment]
Aaaand did I pass this challenge?

**Comment by deltakosh** [comment]
YOU DID :)
Congrats my friend!

**Comment by Dok11** [comment]
Yes! Thanks man!
One contributor more haha =)

**Pull request merged**